helm-charts/yunikorn/templates/deployment.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  replicas: {{ .Values.replicaCount }}
+  replicas: 1
```

Comment:
Hi @craigcondit 

I am just curious about the decision on why the replicas value is now hard coded. Does this feature no longer work with values > 1? Thanks!

Reply:
@elihschiff, it never did (for the scheduler itself), only for the admission controller, which has its own variable. Setting it >1 causes a lot of problems, so it was removed. YuniKorn is designed to recover very rapidly without any shared state, so having two active schedulers is not necessary.

Reply:
Got it, thanks for the quick reply!